package.json

```diff
@@ -26,7 +26,8 @@
     ]
   },
   "scripts": {
-    "build": "turbo run build",
+    "build:packages": "turbo run build --filter=!polaris.shopify.com",
+    "build": "turbo run build --filter=!polaris.shopify.com && turbo run build --filter=polaris.shopify.com",
```

Comment:
The website depends on the built packages, so it's important to run the website build _after_ the package build.